target_bigquery/stream_utils.py

```diff
@@ -105,7 +105,7 @@ def parse_datetime(dt):
             return None
 
     extended_record = record_message['record']
-    extended_record['_sdc_extracted_at'] = parse_datetime(record_message['time_extracted'])
+    extended_record['_sdc_extracted_at'] = parse_datetime(record_message.get('time_extracted')) or datetime.now()
```

Comment:
```suggestion
    extended_record['_sdc_extracted_at'] = parse_datetime(record_message.get('time_extracted', datetime.now()))
```
I'd probably use the default parameter of `get` instead of an `or`